app/models/school_record.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SchoolRecord < ApplicationRecord
+  has_paper_trail
+
+  # Associations
+  belongs_to :school
+  belongs_to :participant_profile, class_name: "ParticipantProfile::ECF", touch: true
+  belongs_to :joining_induction_record, class_name: "InductionRecord"
```

Comment:
Using induction records might be overkill and not required. As it adds extra overhead around linking and maintaining those

Reply:
The opportunity here is that it can include a status specific to a school. If a participant moves through several schools you will know the status of the participant for each school that was responsible for their induction and when